overrides/main.html

```diff
@@ -0,0 +1,10 @@
+<!-- This file has the content for our announcement bar. -->
+<!-- Use target="_blank" when linking to a external site, so the link opens in a new tab. -->
+<!-- https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#announcement-bar -->
+<!-- https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks -->
+
+{% extends "base.html" %}
+
+{% block announce %}
+Read the release notes for our recent major release: <a href="https://github.com/renovatebot/renovate/releases/tag/32.0.0" target="_blank">32.0.0</a>.
```

Comment:
In this case it was pretty long ago, and I don't think we need to show this to everyone forever. Maybe start with v33 and leave it up for a month?

Reply:
> In this case it was pretty long ago, and I don't think we need to show this to everyone forever.

Fair enough. I just wanted to put something in there to test it actually works in the server preview.

> Maybe start with v33 and leave it up for a month?

That's an option, or we can:

1. Nudge our adventurous users to try out `configMigration` so we can get more bugs fixed before general release.
2. Highlight the new Swissquote use case article.

Can you tell me what you want me to do? 😉

Reply:
Pointing to config migration sounds like a good idea, especially once it's a bit more stable. I think we still have closing/reopening problems related to cache though.

Reply:
How about this sequence of announcements:

1. Highlight the Swissquote article first
2. Then once it's stable enough, nudge users to try `configMigration` before general release
3. Highlight any major Renovate version changelog, so users read the breaking changes

```suggestion
Read how Swissquote uses Renovate in the new <a href="https://docs.renovatebot.com/user-stories/swissquote/" target="_blank">Swissquote user story</a> on our site.
```